descartes_samplers/include/descartes_samplers/evaluators/impl/distance_edge_evaluator.hpp

```diff
@@ -85,26 +58,17 @@ bool DistanceEdgeEvaluator<FloatType>::evaluate(const Rung_<FloatType>& from,
                    }
                  });
 
-  // Allocate
-  edges.resize(n_start);
-
-  for (std::size_t i = 0; i < n_start; ++i)
+  FloatType cost = static_cast<FloatType>(0.0);
+  for (std::size_t i = 0; i < delta_thresholds.size(); ++i)
```

Comment:
`delta_thresholds` is confusing to me. It's actually the joint distance limit that each joint can travel to meet the timing constraint, so what does delta mean? We should rename this to something like `joint_distance_threshold`

Reply:
Yep it is joint distance.